scripts/bash/buildInternalDependencies.sh

```diff
@@ -1,22 +1,60 @@
 #!/bin/bash
+
 DIR=`dirname "$0"`
 
 CONCURRENT_BUILD_BATCH_SIZE=2
 
-concurrent_build_commands=()
+USAGE="Usage: buildInternalDependencies.sh [--watch] [--withTypes]"
+
+watch=false
+with_types=false
+while [ "$1" != "" ]; do
+  case $1 in
+      --watch)
+        shift
+        watch=true
+        ;;
+      --withTypes)
+        shift
+        with_types=true
+        shift
+        ;;
+      -h | --help )
+        echo -e "$USAGE\n";
+        exit
+        ;;
+      * )
+        echo -e "$USAGE\n"
+        exit 1
+  esac
+done
+
+[[ $watch = "true" ]] && build_args="--watch" || build_args=""
+[[ $watch = "true" ]] && build_ts_args="--watch --preserveWatchOutput" || build_ts_args=""
+
+build_commands=()
+
+# Build dependencies
 for PACKAGE in $(${DIR}/getInternalDependencies.sh); do
-  concurrent_build_commands+=("\"yarn workspace @tupaia/${PACKAGE} build $1\"") # $1 may pass in --watch
+  build_commands+=("\"yarn workspace @tupaia/${PACKAGE} build $build_args\"")
```

Comment:
@edmofro this variable was renamed in https://github.com/beyondessential/tupaia/commit/9e8c73a2e162d47eccc16180db86db7aaafad462#diff-56f41ce89d253876ed3585b40ae95fc2 from `concurrent_build_commands` to `build_commands`, but still uses the former name in `dev`. Just wanted to confirm that you are fine with this. I personally prefer the new name used here

Reply:
Yep this is good